### PR TITLE
visionfive2: Add NixOS

### PR DESCRIPTION
**Pull request by aisuneko**
- Add NixOS test report for StarFive VisionFive2
- ~~(reports for Milk-V Duo/Duo256M/DuoS are under way)~~

### CODE REVIEW

**Comment by aisuneko** [comment]
Building for Duo/Duo256M failed due to #124. I suppose that we merge the one for VF2 first...

File: VisionFive2/NixOS/README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+## Installation Steps
+
+### Compiling Image
+
+Run `nix build ".#hydraJobs.visionfive2"` if you happen to be using NixOS on your host.
+
+It is not recommended to install and use the `nix` package manager on other distributions. Try using e.g. the official Docker image instead:
+```shell
+docker run -ti -v .:/work ghcr.io/nixos/nix
```
Comment (KevinMX) [inline VisionFive2/NixOS/README.md:38]: Me personally won't use `.` for `docker -v` and I won't recommend it, but anyway, it's up to you.

**Review by KevinMX — APPROVED** [review]

File: VisionFive2/NixOS/README.md (hunk 1 above)
Comment (aisuneko) [reply VisionFive2/NixOS/README.md:38]: > Me personally won't use `.` for `docker -v` and I won't recommend it
Then which one do you usually use? `$(pwd)`?

Comment (KevinMX) [reply VisionFive2/NixOS/README.md:38]: > > Me personally won't use `.` for `docker -v` and I won't recommend it
> 
> Then which one do you usually use? `$(pwd)`?

Yeah, `$(pwd)` LGTM.

Comment (aisuneko) [reply VisionFive2/NixOS/README.md]: Fixed by https://github.com/ruyisdk/support-matrix/pull/123/commits/d45cf3bae489d43c054c72adf10f86e8bee15554

**Pull request merged**